### DISCUSSION TITLE
My McAfee Antivirus is reporitng a virus in compiled 32bit EXE

### DISCUSSION THREAD

**Discussion by YogiYang** [twinbasic/twinbasic#499] [General]
The subject says it all.

- I started VSCode.
- Opened/Created Sample 6.    CustomControls.twinproj
- Clicked on Build button

Any ideas as to why McAfee reports this as a virus?

**Comment by YogiYang** [comment]
![VirusWarningInTwinBasic](https://user-images.githubusercontent.com/99412/142718708-382bdfda-2ed6-4a88-9483-0e21a2d4fdcf.png)
Please check this screen shot for better idea

**Comment by WaynePhillipsEA** [comment]
It has become 'the norm' for anti virus software to flag new / unseen EXEs.  Sometimes they use heuristics, but often they just seem to flag any new/unseen file until they have verified it or when the files becomes more commonly seen.   Signing files with certificates supposedly helps the situation.

The same thing happens to me with Avast and literally any EXEs I generate from Visual Studio, so I have to add the output directory to the exception list in Avast.

In short, there's no virus.  It's just a false positive, and you'll need to adjust your anti-virus software accordingly.

**Reply by wqweto** [reply]
Adjust accordingly here means adding `D:\Job` folder and its subfolders to the exclusion list of folders in McAfee’s interface which would speed up compilation and editor access too.

**Reply by YogiYang** [reply]
> Adjust accordingly here means adding `D:\Job` folder and its subfolders to the exclusion list of folders in McAfee’s interface which would speed up compilation and editor access too.

Thanks for the tip. I know this but in version of McAfee Antivirus I can only add a executable file to exclusion list.

If you know how we can exclude a folder in McAfee please do guide me.

**Reply by YogiYang** [reply]
@WaynePhillipsEA ,

The surprising part is that it does not flag 64Bit executable complied by TB of the same application!
